deploy/swrs/transform/function/load_fuel_charge.sql

```diff
@@ -8,538 +8,126 @@ create or replace function swrs_transform.load_fuel_charge()
+        INSERT INTO swrs_load.fuel_charge (fuel_charge, start_date, end_date, fuel_carbon_tax_details_id, carbon_tax_act_fuel_type_id) VALUES (0.0765, '2017-04-01', '2018-03-31',51, 8);
+        INSERT INTO swrs_load.fuel_charge (fuel_charge, start_date, end_date, fuel_carbon_tax_details_id, carbon_tax_act_fuel_type_id) VALUES (0.0893, '2018-04-01', '2019-03-31',51, 8);
+        INSERT INTO swrs_load.fuel_charge (fuel_charge, start_date, end_date, fuel_carbon_tax_details_id, carbon_tax_act_fuel_type_id) VALUES (0.102, '2019-04-01', '2020-03-31',51, 8);
+        INSERT INTO swrs_load.fuel_charge (fuel_charge, start_date, end_date, fuel_carbon_tax_details_id, carbon_tax_act_fuel_type_id) VALUES (0.1013, '2020-04-01', '2021-03-31',51, 8);
+        INSERT INTO swrs_load.fuel_charge (fuel_charge, start_date, end_date, fuel_carbon_tax_details_id, carbon_tax_act_fuel_type_id) VALUES (0.1127, '2021-04-01', '9999-12-31',51, 8);
+        INSERT INTO swrs_load.fuel_charge (fuel_charge, start_date, end_date, fuel_carbon_tax_details_id, carbon_tax_act_fuel_type_id) VALUES (0.0528, '1899-12-30', '2017-03-31',11, 9);
+        INSERT INTO swrs_load.fuel_charge (fuel_charge, start_date, end_date, fuel_carbon_tax_details_id, carbon_tax_act_fuel_type_id) VALUES (0.0528, '2017-04-01', '2018-03-31',11, 9);
+        INSERT INTO swrs_load.fuel_charge (fuel_charge, start_date, end_date, fuel_carbon_tax_details_id, carbon_tax_act_fuel_type_id) VALUES (0.0616, '2018-04-01', '2019-03-31',11, 9);
+        INSERT INTO swrs_load.fuel_charge (fuel_charge, start_date, end_date, fuel_carbon_tax_details_id, carbon_tax_act_fuel_type_id) VALUES (0.0704, '2019-04-01', '2020-03-31',11, 9);
+        INSERT INTO swrs_load.fuel_charge (fuel_charge, start_date, end_date, fuel_carbon_tax_details_id, carbon_tax_act_fuel_type_id) VALUES (0.0704, '2020-04-01', '2021-03-31',11, 9);
+        INSERT INTO swrs_load.fuel_charge (fuel_charge, start_date, end_date, fuel_carbon_tax_details_id, carbon_tax_act_fuel_type_id) VALUES (0.089, '2021-04-01', '9999-12-31',11, 9);
+        INSERT INTO swrs_load.fuel_charge (fuel_charge, start_date, end_date, fuel_carbon_tax_details_id, carbon_tax_act_fuel_type_id) VALUES (0.0483, '1899-12-30', '2017-03-31',19, 10);
+        INSERT INTO swrs_load.fuel_charge (fuel_charge, start_date, end_date, fuel_carbon_tax_details_id, carbon_tax_act_fuel_type_id) VALUES (0.0483, '2017-04-01', '2018-03-31',19, 10);
+        INSERT INTO swrs_load.fuel_charge (fuel_charge, start_date, end_date, fuel_carbon_tax_details_id, carbon_tax_act_fuel_type_id) VALUES (0.0564, '2018-04-01', '2019-03-31',19, 10);
+        INSERT INTO swrs_load.fuel_charge (fuel_charge, start_date, end_date, fuel_carbon_tax_details_id, carbon_tax_act_fuel_type_id) VALUES (0.0644, '2019-04-01', '2020-03-31',19, 10);
+        INSERT INTO swrs_load.fuel_charge (fuel_charge, start_date, end_date, fuel_carbon_tax_details_id, carbon_tax_act_fuel_type_id) VALUES (0.0315, '2020-04-01', '2021-03-31',19, 10);
```

Comment:
Let's add a comment here:
> Because of a methodology change that started in 2020, the Coke Oven Gas fuel charge decreased, which meant that the rate change was not delayed with the Covid-19 emergency regulation. Therefore this fuel does not need for a separate rate for 2021-2022 vs 2022 onward

Reply:
This comment could be in a column, so that this information is visible to both developers and users of the database.